apps/platform_api/src/modules/exchangeKey/exchangeKey.resolver.ts

```diff
@@ -13,9 +16,13 @@ export class ExchangeKeyResolver {
     return await this.exchangeKeyService.find(id);
   }
 
-  @Mutation(() => Boolean, { description: 'Create exchange key' })
-  async createExchangeKey(@Args('input') input: CreateExchangeKeyInput): Promise<boolean> {
-    return await this.exchangeKeyService.create(input);
+  @Mutation(() => Result, { description: 'Create exchange key' })
+  @UseGuards(GqlAuthGuard)
```

Comment:
you probably want guard on this entire resolver yeah?

Reply:
yes! moved this guard for the entire resolver